### DISCUSSION TITLE
Github Visualizations

### DISCUSSION THREAD

**Discussion by tecunningham** [jotaen/klog#58] [Show and tell]
Probably this is obvious, but just occured to me that this would be a great visualization of the klog data:

<img width="852" alt="image" src="https://user-images.githubusercontent.com/14047958/113784764-d1204a00-96ea-11eb-8f17-0bb38f6b40df.png">

I will try to do this in R sometime, but also occurred to me it would be cool to do  in the terminal: e.g. each cell can be a number representing number of hours recorded on that day, it would be a very efficient way of representing the total work on a long-running project.

**Comment by jotaen** [comment]
In the terminal I also experimented with simple text-based status bars for the `report` command, along the lines of this:

```
Mo,  8th ▬▬▬▬▬▬▬▬▬▬▬▬▬▬
Tu,  9th ▬▬▬▬▬
We, 10th ▬▬▬▬▬▬▬▬▬▬▬▬
Fr, 12th ▬▬
```

It’s very limited in terms of visual precision, though (or at least I didn’t find a good way), so I didn’t experiment any further.
